src/components/NotificationError/NotificationError.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import {
+  FC,
+  useEffect,
+} from 'react';
+
+import cn from 'classnames';
+
+type Props = {
+  error: string;
+  resetError: () => void;
+};
+
+const NotificationError: FC<Props> = ({ error, resetError }) => {
+  useEffect(() => {
+    setTimeout(() => {
```

Comment:
Clear all timers on the component unmount
https://bobbyhadz.com/blog/react-cleartimeout